src/Sql/Sql/ServiceObjective/Cmdlet/GetAzureSqlServerServiceObjective.cs

```diff
@@ -44,7 +45,11 @@ protected override IEnumerable<AzureSqlServerServiceObjectiveModel> GetEntity()
         {
             ICollection<AzureSqlServerServiceObjectiveModel> results = null;
 
-            if (this.MyInvocation.BoundParameters.ContainsKey("ServiceObjectiveName") && !WildcardPattern.ContainsWildcardCharacters(ServiceObjectiveName))
+            if (this.ParameterSetName == ByLocationNameParameterSet)
```

Comment:
should filtering by SLO name also be supported with LocationName param set?

Reply:
Yes, done